snapcraft/_store.py

```diff
@@ -389,8 +409,18 @@ def push(snap_filename, release_channels=None):
 
     If release_channels is defined it also releases it to those channels if the
     store deems the uploaded snap as ready to release.
+
+    If only_metadata is True it will just send the metadata; otherwise the
+    whole process is done (including sending the metadata after sending the
+    snap binary).
```

Comment:
Can you please document the force_metadata arg?

Reply:
Done, thanks!